onlineweb4/views.py

```diff
@@ -1,21 +1,22 @@
 # -*- coding: utf-8 -*-
 import logging
-from smtplib import SMTPException
 
 from django.conf import settings
 from django.contrib import messages
 from django.core.mail import send_mail
 from django.shortcuts import redirect, render
 from onlineweb4.forms import ErrorForm
+from smtplib import SMTPException
 
 
 def server_error(request):
     log = logging.getLogger(__name__)
 
     if request.method == 'POST':
-        form = ErrorForm(request.POST)
-        message = request.POST.get('reason', 'Ingen forklaring oppgitt.')
-        try: 
+        message = request.POST.get('reason')
+        if message is None:
```

Comment:
It would suffice to do `if message:`, since `if message:` would call `bool()` on `message`, resulting in `bool(message)` => `True` for anything other than `""` (empty string), which would be `False`. 

So if an empty form is submitted, or nothing is submitted, this would be `False`, resulting in a redirect to home.

TL;DR: `if message:` (instead of `if message is None:`) would also catch an empty form.